plugins/rtmp-services/data/services.json

```diff
@@ -779,6 +779,16 @@
                 "bframes": 0
             }
         },
+        {
+          "name": "GameTips.TV - Service Stream Game Iran",
+          "common": true,
```

Comment:
As this is not a common service, this line should be omitted.

Reply:
This service is special for Iran